spec/api_callers/http_request_spec.rb

```diff
@@ -2,22 +2,36 @@
 require 'net/http'
 
 describe ::ApiCallers::HttpRequest do
-  subject { ::ApiCallers::HttpRequest.new('') }
+  subject { ::ApiCallers::HttpRequest.new('some_uri', http_verb) }
+  let(:http_verb) { 'get' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.